src/components/Popover/Popover.module.css

```diff
@@ -0,0 +1,137 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # POPOVER
+\*------------------------------------*/
+
+/**
+ * 1) Box that contains content
+ */
+.popover {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-start;
+  visibility: hidden;
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: var(--eds-z-index-200) 0;
+  background-color: transparent;
+  transition: all var(--eds-anim-fade-long) var(--eds-anim-ease);
+  transform: translateZ(0);
+
+  &.eds-is-active {
+    visibility: visible;
+  }
+
+  &:focus-visible {
+    @mixin focus;
+  }
+}
+
+/**
+ * Popover window
+ * 1) Actual box with content within
+ */
+.popover__window {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: auto;
+  left: 0;
+  bottom: 68px;
+  height: auto;
+  max-height: 450px;
+  width: 400px;
+  max-width: 400px;
+  margin-left: 13px;
+  opacity: 0;
+  background: var(--eds-color-neutral-white);
+  border-radius: var(--eds-theme-border-radius);
+  box-shadow: var(--eds-theme-box-shadow);
+  transition: opacity var(--eds-anim-fade-quick) var(--eds-anim-ease);
+  overscroll-behavior-y: contain;
+
+  &:focus-visible {
+    @mixin focus;
+  }
+
+  /**
+  * Popover arrow
+  * 1) A 0 height/0 width empty container positioned at the bottom of the popover; its purpose is to provide...
+  * 2) ...a psuedo element that adds a triangle added to the bottom edge of the popover pointing toward the button that opened the popover
+  */
+  .popover__arrow {
+    position: relative;
+    bottom: 0;
+    left: 0;
+  }
+  &::after {
+    /* 2 */
+    position: absolute;
+    left: 20px;
+    top: 0;
+    border-top-color: var(--eds-color-neutral-white);
+    border-left: 15px solid transparent;
+    border-right: 15px solid transparent;
+    border-style: solid;
+    content: '';
+    border-width: 10px;
+    border-bottom-width: 0;
+  }
+
+  /**
+   * Popover window inside active popover
+   * 1) Transform added for animation
+   */
+  .popover.eds-is-active & {
+    opacity: 1;
+    transform: translateY(0);
+    transition: opacity var(--eds-anim-fade-quick) var(--eds-anim-ease);
+  }
+}
+
+/**
+* Popover content
+* 1) Includes header, body and footer of popover
+*/
+.popover__content {
+  padding-top: var(--eds-size-2);
+  padding-bottom: var(--eds-size-2);
+  height: auto;
+  max-height: 25rem;
+  overflow-y: auto;
+}
+
+/**
+ * Popover header
+ */
+.popover__header {
+  margin-bottom: var(--eds-size-2);
+
+  h2 {
```

Comment:
This `h2` ruleset can be removed since you're using a heading variation. These styles will tighten up as we import our updated typography presets. Thanks!